keyboards/handwired/dactyl_manuform/4x6/keymaps/via/keymap.c

```diff
@@ -0,0 +1,110 @@
+/*
+This is the c configuration file for the keymap
+
+Copyright 2012 Jun Wako <wakojun@gmail.com>
+Copyright 2015 Jack Humbert
+Copyright 2023 André Büsgen <andre.buesgen@posteo.de>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#include QMK_KEYBOARD_H
+
+#define _BASE 0
+#define _RAISE 1
+#define _LOWER 2
+#define _BLANK 3
```

Comment:
```suggestion
enum layers{
    _BASE,
    _RAISE,
    _LOWER,
    _BLANK
};
```
As per the [`docs`](https://docs.qmk.fm/#/pr_checklist?id=keymap-prs) for Keymap PRs: 
> prefer layer `enum`s to `#defines`